Source/missiles.cpp

```diff
@@ -557,11 +560,13 @@ void CheckMissileCol(int i, int mindam, int maxdam, bool shift, Point position,
 			}
 		}
 	} else {
-		if (dMonster[mx][my] > 0) {
+		int mid = dMonster[mx][my];
+		if (mid > 0) {
+			mid -= 1;

```

Comment:
Inconsistent with above, for style purposes would be good to use the same code 😄